Libraries/CMSIS/Device/Maxim/MAX32690/Include/ptg_regs.h

```diff
@@ -147,47 +187,155 @@ typedef struct {
 
+#define MXC_F_PTG_RESYNC_PT4_POS                       4 /**< RESYNC_PT4 Position */
+#define MXC_F_PTG_RESYNC_PT4                           ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT4_POS)) /**< RESYNC_PT4 Mask */
+
+#define MXC_F_PTG_RESYNC_PT5_POS                       5 /**< RESYNC_PT5 Position */
+#define MXC_F_PTG_RESYNC_PT5                           ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT5_POS)) /**< RESYNC_PT5 Mask */
+
+#define MXC_F_PTG_RESYNC_PT6_POS                       6 /**< RESYNC_PT6 Position */
+#define MXC_F_PTG_RESYNC_PT6                           ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT6_POS)) /**< RESYNC_PT6 Mask */
+
+#define MXC_F_PTG_RESYNC_PT7_POS                       7 /**< RESYNC_PT7 Position */
+#define MXC_F_PTG_RESYNC_PT7                           ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT7_POS)) /**< RESYNC_PT7 Mask */
+
+#define MXC_F_PTG_RESYNC_PT8_POS                       8 /**< RESYNC_PT8 Position */
+#define MXC_F_PTG_RESYNC_PT8                           ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT8_POS)) /**< RESYNC_PT8 Mask */
+
+#define MXC_F_PTG_RESYNC_PT9_POS                       9 /**< RESYNC_PT9 Position */
+#define MXC_F_PTG_RESYNC_PT9                           ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT9_POS)) /**< RESYNC_PT9 Mask */
+
+#define MXC_F_PTG_RESYNC_PT10_POS                      10 /**< RESYNC_PT10 Position */
+#define MXC_F_PTG_RESYNC_PT10                          ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT10_POS)) /**< RESYNC_PT10 Mask */
+
+#define MXC_F_PTG_RESYNC_PT11_POS                      11 /**< RESYNC_PT11 Position */
+#define MXC_F_PTG_RESYNC_PT11                          ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT11_POS)) /**< RESYNC_PT11 Mask */
+
+#define MXC_F_PTG_RESYNC_PT12_POS                      12 /**< RESYNC_PT12 Position */
+#define MXC_F_PTG_RESYNC_PT12                          ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT12_POS)) /**< RESYNC_PT12 Mask */
+
+#define MXC_F_PTG_RESYNC_PT13_POS                      13 /**< RESYNC_PT13 Position */
+#define MXC_F_PTG_RESYNC_PT13                          ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT13_POS)) /**< RESYNC_PT13 Mask */
+
+#define MXC_F_PTG_RESYNC_PT14_POS                      14 /**< RESYNC_PT14 Position */
+#define MXC_F_PTG_RESYNC_PT14                          ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT14_POS)) /**< RESYNC_PT14 Mask */
+
+#define MXC_F_PTG_RESYNC_PT15_POS                      15 /**< RESYNC_PT15 Position */
+#define MXC_F_PTG_RESYNC_PT15                          ((uint32_t)(0x1UL << MXC_F_PTG_RESYNC_PT15_POS)) /**< RESYNC_PT15 Mask */
+
 /**@} end of group PTG_RESYNC_Register */
 
 /**
  * @ingroup  ptg_registers
- * @defgroup PTG_INTFL PTG_INTFL
+ * @defgroup PTG_STOP_INTFL PTG_STOP_INTFL
  * @brief    Pulse Train Interrupt Flags
  * @{
  */
-#define MXC_F_PTG_INTFL_PT0_POS                        0 /**< INTFL_PT0 Position */
```

Comment:
Same comments as other reg files - redirect these but only if the generator can do it.